package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "aqovia-vscode-extension-pack",
+  "displayName": "Aqovia VSCode ExtensionPack",
+  "description": "Aqovia Visual Studio Code Extension Pack",
+  "version": "1.3.34",
+  "publisher": "Aqovia",
+  "license": "SEE LICENSE IN LICENSE.txt",
+  "engines": {
+    "vscode": "^1.80.0"
+  },
+  "scripts": {
+    "pack": "vsce package --out VSCodeExtensionsPack.vsix"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Aqovia/VisualStudioCode.ExtensionPack"
+  },
+  "categories": [
+    "Extension Packs"
+  ],
+  "extensionPack": [
+    "hnw.vscode-auto-open-markdown-preview",
+    "redhat.vscode-yaml",
+    "Zignd.html-css-class-completion",
+    "streetsidesoftware.code-spell-checker",
+    "ms-dotnettools.csharp",
```

Comment:
This is base language support. I think we should instead be importing `ms-dotnettools.csdevkit` (which is a pack of official Microsoft extensions for C# language support).

Reply:
Changed.